src/store/actions/onboarding.actions.ts

```diff
@@ -44,6 +47,32 @@ export const finalizeOnboarding = (): ThunkAction<Promise<void>, RootState, unkn
             identifier: contactIdentifier
           })
         )
+        dispatch(
+          createVerifiableCredential({
+            credential: {
+              "@context": [
+                "https://www.w3.org/2018/credentials/v1",
+                "https://www.w3.org/2018/credentials/examples/v1",
+                "https://sphereon-opensource.github.io/ssi-mobile-wallet/context/sphereon-wallet-identity-v1.jsonld"
+              ],
+              id: uuidv4(),
+              type: [
+                "VerifiableCredential",
+                "SphereonWalletIdentity"
+              ],
+              issuer: identifier.did,
+              issuanceDate: new Date(),
+              credentialSubject: {
+                id: identifier.did,
+                SphereonWalletIdentity: {
```

Comment:
I would put the names/email directly in credential subject. No need for an aditional object